domains/auth/Auth.API/API/Services/OidcProviders/IOidcProviders.cs

```diff
@@ -1,7 +1,8 @@
 using API.Models;
 
-namespace API.Services;
+namespace API.Services.OidcProviders;
 public interface IOidcProviders
 {
     NextStep CreateAuthorizationUri(AuthState state);
+    Task Logout(string token);
```

Comment:
This interface contains two methods: one that is async and one that is not. I can see two ways of solving this:

* Rename `Logout` to `LogoutAsync`, so it is clear that this is async
* Change `CreateAuthorizationUri` to also be async. Which mean that it returns Task<NextStep>